packages/utils/interactive-hooks/src/useAsPressable.ts

```diff
@@ -1,19 +1,23 @@
 import * as React from 'react';
+import { PressableProps, Platform } from 'react-native';
+import { PressableFocusProps, PressableHoverProps, PressablePressProps } from './Pressability/Pressability.types';
 import {
   IPressableHooks,
   IWithPressableOptions,
   IPressState,
   IHoverState,
   IFocusState,
   IWithPressableEvents,
+  IPressableState,
+  PressablePropsExtended,
 } from './useAsPressable.types';
 import { usePressability } from './usePressability';
 
 /**
  * hover specific state and callback helper
  */
 // eslint-disable-next-line @typescript-eslint/ban-types
-function useHoverHelper<T extends object>(props: IWithPressableOptions<T>): [IWithPressableOptions<object>, IHoverState] {
+function useHoverHelper(props: PressableHoverProps): [PressableHoverProps, IHoverState] {
```

Comment:
Do we still need the aslant disable comments here?

Reply:
Same for below lines.

Reply:
Good catch